storybook/pages/CommunityCollectibleViewPage.qml

```diff
@@ -24,7 +24,6 @@ SplitView {
 
                 anchors.fill: parent
                 anchors.margins: 50
-                holdersModel: TokenHoldersModel {}
```

Comment:
Is this still needed then, can't we remove it?

Reply:
It's not a property of `CommunityCollectibleView` but the model is already used to test other views!